includes/class-lipscore-widget-manager.php

```diff
@@ -87,6 +93,18 @@ public function add_html_to_questions_tab_title( $title ) {
         return $this->render_widget( 'questions_title' );
     }
 
+    public function add_testimonial() {
+        return $this->render_widget( 'testimonial_widget' );
```

Comment:
Please remove the `_widget` postfix here as well. Probably it also will produce an error because widget template is named 'testimonial.php' now.

Reply:
Oops, my bad :)